README.md

```diff
@@ -143,19 +144,33 @@ In `sites/index.js`, locate `secret` and change `CHANGEME` to a random string of
 
 ### Operating System: Mac, Linux, or Virtual Linux
 
-**Your local development environment must be either MacOS or Linux.** If your development computer runs Windows, we recommend development on Ubuntu Linux in a full virtual Linux machine, via [VirtualBox](https://www.virtualbox.org/).
-
-Another option is to use the Windows Subsystem for Linux, which is also an Ubuntu Linux-based environment. However this option has not been extensively tested with Assembly.
+**Your local development environment must be either MacOS or Linux.** If your development computer runs Windows, we recommend development on Ubuntu Linux in a full virtual Linux machine. This can be through [WSL2 (Windows Subsystem for Linux)](https://learn.microsoft.com/en-us/windows/wsl/install) or via [VirtualBox](https://www.virtualbox.org/).
 
 ### Software Installation Requirements
 
 To test-drive the project in development, make sure you have Apostrophe's usual dependencies on your local machine:
 
-* MongoDB (4.4.x or better)
-* NodeJS (14.x or better, latest long term support release recommended)
-* Imagemagick (for fast, high-quality image rendering)
+* MongoDB (5.0 or better, we recommend 6.0)
```

Comment:
5.0 is past its end of life date. We should say 6.0 or better, recommend 8.0

We do test 8.0

Reply:
Updated.